apps/api/src/app/inbox/usecases/update-preferences/update-preferences.spec.ts

```diff
@@ -64,14 +66,16 @@ describe('UpdatePreferences', () => {
     subscriberPreferenceRepositoryMock = sinon.createStubInstance(SubscriberPreferenceRepository);
     getSubscriberGlobalPreferenceMock = sinon.createStubInstance(GetSubscriberGlobalPreference);
     getSubscriberTemplatePreferenceUsecase = sinon.createStubInstance(GetSubscriberTemplatePreference);
+    upsertPreferenceMock = sinon.createStubInstance(UpsertPreferences);
 
     updatePreferences = new UpdatePreferences(
       subscriberPreferenceRepositoryMock as any,
       notificationTemplateRepositoryMock as any,
       subscriberRepositoryMock as any,
       analyticsServiceMock as any,
       getSubscriberGlobalPreferenceMock as any,
-      getSubscriberTemplatePreferenceUsecase as any
+      getSubscriberTemplatePreferenceUsecase as any,
+      upsertPreferenceMock as any
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** Unrelated to my changes but my editor was (rightfully) complaining about an incorrect number of dependencies here‏